pysuccinct/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+PYTHON_ADD_MODULE(pycompress pycompress.cpp)
+FILE(COPY pycompress.py DESTINATION .)
+
+PYTHON_ADD_MODULE(pyquery_file pyquery_file.cpp)
+FILE(COPY pyquery_file.py DESTINATION .)
+
+PYTHON_ADD_MODULE(pyquery_kv pyquery_kv.cpp)
+FILE(COPY pyquery_kv.py DESTINATION .)
+
+PYTHON_ADD_MODULE(pyquery_semistructured pyquery_semistructured.cpp)
+FILE(COPY pyquery_semistructured.py DESTINATION .)
+
+cmake_minimum_required(VERSION 2.8)
+project(succinct-pysuccinct CXX)
+
+include(CheckCXXCompilerFlag)
```

Comment:
Since this check is already performed in CMakeLists.txt in the main file, it is unnecessary here. (Lines 16-28)